qa/L0_config_search/test.sh

```diff
@@ -28,7 +28,7 @@ EXPORT_PATH="`pwd`/results"
 FILENAME_SERVER_ONLY="server-metrics.csv"
 FILENAME_INFERENCE_MODEL="model-metrics-inference.csv"
 FILENAME_GPU_MODEL="model-metrics-gpu.csv"
-TRITON_LAUNCH_MODES="remote local"
+TRITON_LAUNCH_MODES="local remote"
```

Comment:
any reason why you chose L0_config_search instead of L0_server_launch_modes?

Reply:
I couldn't easily figure out how to change that test and this gets us most of what we need (local and remote).